frontend/src/utils/interpreter/findFunction.ts

```diff
@@ -50,35 +50,28 @@ function findAll(this: any | undefined, arg1: any, arg2: any): any {
 (Object.prototype as any).find = find;
 (Object.prototype as any).findAll = findAll;
 
-
-
-
 // calls findByName or findByCallback depending on the arguments
 function findGeneric(self: any, arg1: any, arg2: any, returnFirstResult: boolean): any[] {
-
     const ignoreCase = Boolean(arg2);
     const caseSensitive = !ignoreCase;
 
     if (typeof arg1 == 'string') {
         // findByName
         const propertyName = String(arg1);
         return findByName(self, propertyName, caseSensitive, returnFirstResult);
-    }
-    else if (typeof arg1 == 'function') {
+    } else if (typeof arg1 == 'function') {
```

Comment:
I liked the extra line break. It helped to indicate that a new block is starting.
If we have this rule, I would like it to add a line break before any `else if` blocks.

Reply:
I see this, Gonna try to find a way to enforce it. and add it if possible, I think in this case my prettier settings did the change.